README.md

```diff
@@ -88,6 +88,7 @@ To install Mender on a device from source, please run the following commands
 inside the cloned repository:
 
 ```
+go get
```

Comment:
This will download the dependencies instead of using the vendored ones. It will most likely "work", but it is not what we want.

Instead, the user should clone the repo in an specific path inside its own GOPATH. I'll launch a new PR with the recommended steps to clarify my point.

Reply:
See #474